pegasus/tools/visualization.py

```diff
@@ -136,18 +143,22 @@ def calc_force_directed_layout(
     TODO: Typing
     """
     G = construct_graph(W)
+    try:
+        import forceatlas2 as fa2
+    except PackageNotFoundError:
+        print("Please install module forceatlas2")
```

Comment:
1. Correct the package name to `forceatlas2-python`.
2. Add installation suggestion `pip install forceatlas2-python`.